src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,22 @@
+.stars {
+  margin: 17px 0;
+  display: flex;
+
+  &__star {
+    height: 16px;
+    width: 16px;
+    margin-right: 4px;
+    border-radius: 0.5px;
+    background-image: url(../../src/images/star.svg);
+    background-repeat: no-repeat;
+    background-position: center;
+  }
+}
+
+.stars--5 .stars__star,
+.stars--1 .stars__star:first-child,
+.stars--2 .stars__star:nth-child(-n + 2),
+.stars--3 .stars__star:nth-child(-n + 3),
+.stars--4 .stars__star:nth-child(-n + 4) {
+  background-image: url(../../src/images/star-active.svg);
+}
```

Comment:
try to use cycle for
more detail in [video](https://mate.academy/learn/css-advanced/sass#/video/926)